packages/features/CalendarEventBuilder.ts

```diff
@@ -226,8 +226,16 @@ export class CalendarEventBuilder {
 
     // Team & calendars
     if (eventType.team) {
+      // We need to get the team members assigned to the booking
+      // In the DB team members are stored in the Attendee table
+      const bookingAttendees = booking.attendees;
+
+      const hostsToInclude = eventType.hosts.filter((host) =>
+        bookingAttendees.some((attendee) => attendee.email === host.user.email)
+      );
```

Comment:
Team members are stored in the `Attendee` table. To prevent any edge cases where fixed hosts change, we should get the team members that are associated with the booking. Not the state of the event type at the time of workflow being sent.